### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by marios-pz** [UniWa-Community/eclass-tui#1]
# TUI

- [x] per run cache
- [x] hide/unhide excluded courses, add courses to config as excluded
- [x] greek keyboard keybinds
- [ ] login + choose domain
- [ ] Config handling?

# Blocked on:

- configuration exporting

**Comment by cultab** [comment]
1, 3 and 5 are already done :P
4 is blocked on upstream library (see https://github.com/UniWa-Community/eclass-utils/pull/1) making configs exportable (or we can hack around it)
2 is next

**Comment by marios-pz** [comment]
BUGS:
- [x] X,C are crashing the channels when value is nil
- [ ] Login Box shows up on every launch
